lib/src/models/phone_metadata_lengths.dart

```diff
@@ -2,28 +2,60 @@ import 'dart:convert';
 
 class PhoneMetadataLengths {
   final List<int> general;
-  final List<int> mobile;
-  final List<int> fixedLine;
+  final List<int>? mobile;

```

Comment:
Do you have any idea if this will be tree shaken, this was one of the big worries for not implementing this at first, to keep the library as small as possible for client usage. 

Maybe this is tree shaken though ?